commands/deploy_command.go

```diff
@@ -82,7 +83,10 @@ func (c *DeployCommand) GetPluginCommand() plugin.Command {
    cf deploy MTA [-e EXT_DESCRIPTOR[,...]] [-t TIMEOUT] [--version-rule VERSION_RULE] [-u URL] [-f] [--retries RETRIES] [--no-start] [--namespace NAMESPACE] [--delete-services] [--delete-service-keys] [--delete-service-brokers] [--keep-files] [--no-restart-subscribed-apps] [--do-not-fail-on-missing-permissions] [--abort-on-error] [--verify-archive-signature] [--strategy STRATEGY] [--skip-testing-phase]
 
    Perform action on an active deploy operation
-   cf deploy -i OPERATION_ID -a ACTION [-u URL]`,
+   cf deploy -i OPERATION_ID -a ACTION [-u URL]
+
+   (EXPERIMENTAL) Deploy a multi-target app archive referenced by a remote URL
+   cf deploy <URL> [-e EXT_DESCRIPTOR[,...]] [-t TIMEOUT] [--version-rule VERSION_RULE] [-u URL] [--retries RETRIES] [--no-start] [--namespace NAMESPACE] [--delete-services] [--delete-service-keys] [--delete-service-brokers] [--keep-files] [--no-restart-subscribed-apps] [--do-not-fail-on-missing-permissions] [--abort-on-error] [--verify-archive-signature] [--strategy STRATEGY] [--skip-testing-phase]`,
```

Comment:
the optional param `URL` here is also ambiguous, because there is an option `[-u URL]`

Reply:
What about the `-f` option? Or it will be set by default?

Reply:
The first `<URL>`  parameter renamed to `<MTA archive URL>`.
The second `[-u URL]` parameter renamed to `[-u MTA_CONTROLLER_URL]`.
In this scenario the `-f` flag doesn't do anything because there is no check for an ongoing operation with the MTA that is being deployed